app/tests/v2/system/test_records.py

```diff
@@ -1,5 +1,19 @@
+import json
 from app.tests.v2.base_test import BaseTest
 
 
 class RecordTest(BaseTest):
-    pass
+    def test_create_new_incident(self):
+        """
+        Test create new incident POST Endpoint.
+        :return: status code 201 (Created)
+        """
+        r = self.create_incident()
+        expected = {"status": 201,
```

Comment:
Let's expect that when a new incident is created, the user receives all the details about it instead of just ID.

Reply:
ok.